modules/repository/main_test.go

```diff
@@ -8,6 +8,7 @@ import (
 
 	"code.gitea.io/gitea/models/unittest"
 
+	_ "code.gitea.io/gitea/models"
```

Comment:
:raised_eyebrow:

Reply:
A better change because that means the `repository` module will not depend on `models` directly. So that the test package needs to depend on `models` explicitly.